ComSemApp/templates/ComSemApp/student/assessment_results.html

```diff
@@ -1,5 +1,72 @@
 {% extends 'ComSemApp/sidebar.html' %}
 
+{% load static %}
+
 {% block content %}
-<h1>Speaking Practice Results View</h1><a href="{% url 'student:speaking_practice_generator' course.id %}">Generator</a>
-{% endblock %}
\ No newline at end of file
+<div class="row h-100">
+	<div class="col-12 h-100">
+        <!-- invisible until setup is complete -->
+		<div class="card-box">
+			<h4 class="header-title m-t-0 m-b-20">Verbal Practice</h4>
+			<!-- bar displaying student's progress throughout the reviewsheet -->
+			<div class="progress"  style="height: 20px;">
+  				<div class="progress-bar bg-success" role="progressbar" style="height: 20px; width: 100%" aria-valuenow="100" aria-valuemin="100" aria-valuemax="100"></div>
+			</div>
+			<div class="card">
+                <div class='card-body'> <!--card for displaying results-->
+                    <h1 class="card-title">Review complete!</h1>
+                    <table id="results_table" class="table table-hover"> <!--table for displaying results of a worksheet-->
+                        <thead>
+                            <tr>
+                                <th>Student Transcription</th> <!--display term that students saw in question-->
+                                <th class="text-center">Accuracy Score</th> <!--displays if term is gramatically correct-->
+                                <th class="text-center">Fluency Score</th> <!--displays if the student's answer was correct-->
+								<th class="text-center invisible"><button class="show-more" type="button">Show More</button></th>
```

Comment:
This button class should also include a button style from [Bootstrap](https://getbootstrap.com/docs/4.0/components/buttons/)